desktop/conf.dist/hue.ini

```diff
@@ -1667,6 +1667,8 @@ submit_to=True
 ## Query sent when no term is entered
 ## empty_query=*:*
 
+## Download limit with max of 15k
+## download_limit=1000
```

Comment:
I think if we increase the limit then might be some performance issue so looks like we should add   "Setting this higher than the default can degrade performance" in comment.